lib/feature/add_seed/enter_seed_phrase/view/enter_seed_phrase_view.dart

```diff
@@ -143,10 +143,10 @@ class EnterSeedPhraseView extends StatelessWidget {
               validator: (v) => v?.isEmpty ?? true ? '' : null,
               needClearButton: false,
               prefixIcon: Padding(
-                // ignore: no-magic-number
                 padding: EdgeInsets.only(
                   left: DimensSize.d12,
-                  top: StyleRes.addRegular.fontSize!,
+                  // ignore: no-magic-number
+                  top: StyleRes.addRegular.fontSize! + 2,
```

Comment:
Imho we shouldn't allow designers to use not uikit-defined things at all.

Reply:
It's not uikit problem, it's some strange bug on our side with BoxConstraints, but ill see

Reply:
-